userguide/analytics/userguide_analytics.adoc

```diff
@@ -1,63 +1,595 @@
-= Kill Bill analytics guide
+= Kill Bill Analytics Guide
 
-This is the reference guide to get started with https://github.com/killbill/killbill-analytics-plugin[Kill Bill analytics plugin].
+The https://github.com/killbill/killbill-analytics-plugin[Kill Bill analytics plugin] provides business analytics and reporting capabilities. You can use this plugin to create reports and view them via Kaui. Not only that, it allows creating reports against external databases as well. This document explains how to install and use this plugin. 
+
+== Prerequisites
+
+* Ensure that you have Kill Bill, Kaui, and the database set up as explained in the https://docs.killbill.io/latest/getting_started.html[Getting Started Guide].
+
+* Ensure that you have https://curl.haxx.se/[cURL] installed. If you are on Windows, we recommend that you use https://git-scm.com/download/win[Git Bash] to run the `cURL` commands.
 
 
 [[components-analytics-overview]]
-== Overview
+== Analytics Plugin Internals
+
+Kill Bill maintains a separate <<analytics_tables, reporting database>> for analytics purposes. The tables in this database are meant to be used as building blocks for reports. In contrary to other tables in the system, this set of tables is stable and potentially changes only between major revisions of Kill Bill. This database is kept current in realtime. The https://github.com/killbill/killbill-analytics-plugin[analytics plugin] is a https://docs.killbill.io/latest/notification_plugin.html[notification plugin]. It listens to https://docs.killbill.io/latest/kill_bill_events.html[events] generated by Kill Bill when any system change occurs and rebuilds the analytics tables upon receipt of events. So, for example, if a new account is created, Kill Bill triggers an `ACCOUNT_CREATION` event. The analytics plugin listens to this event and on receipt updates the analytics tables based on the Kill Bill tables. 
+
+== Plugin Installation
+
+A plugin can be installed via the https://github.com/killbill/killbill-cloud/blob/master/kpm[Kill Bill Package Manager (KPM)]. For this, please follow the steps given below:
+
+. Ensure that you have kpm installed as explained https://github.com/killbill/killbill-cloud/tree/master/kpm#kpm-installation[here].
+
+. Install the plugin using the following `kpm` command (Replace `<path_to_install_plugin>` with the path where you want to install the plugin. This path should match the path specified by the `org.killbill.osgi.bundle.install.dir` property in the  https://docs.killbill.io/latest/userguide_configuration.html#_configuration_properties_table[Kill Bill Configuration]):
```

Comment:
```suggestion
. Install the plugin using the following `kpm` command (Replace `<path_to_install_plugin>` with the path where you want to install the plugin. This path should match the path specified by the `org.killbill.osgi.bundle.install.dir` property in the  https://docs.killbill.io/latest/userguide_configuration.html#_configuration_properties_table[__Kill Bill and Kaui Configuration Guide__]):
```

Reply:
The double underscore ( __ ) around the guide title is necessary to render italics correctly when used with the link URL.

Reply:
Fixed